playground/src/auto-importer/auto-importer.js

```diff
@@ -0,0 +1,245 @@
+// Copyright 2018 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import CodeMirror from 'codemirror';
+
+const MISSING_ENGINE_PARAM = 'amphtml engine v0.js script';
+const AMP_BASE_URL = '"https://cdn.ampproject.org/v0.js"';
```

Comment:
How hard would it be to also support auto-adding the runtime for amp ads?

Reply:
Sure! But only if it doesn't take too much effort.

Reply:
I've added this.